### PR TITLE
Tgui example update

### PR DESCRIPTION
**Pull request by remove32**
## About The Pull Request
I was unsatisfied with the existing documentation, so I updated it with some information that I came up with while learning the framework myself.

## Why It's Good For The Game
Clarified documentation helps new developers get onboarded with the UI framework.

## Changelog
:cl:
 spellcheck: Updated TGUI onboarding documentation
/:cl:

### CODE REVIEW

**Comment by stylemistake** [comment]
Formatting in your javascript examples is broken and linter will scream.

File: tgui/docs/tutorial-and-examples.md [hunk 1]
```diff
@@ -34,32 +36,36 @@ powerful interactions for embedded objects or remote access.
...
 ### Backend
 
-Let's start with a very basic hello world.
+Let's start with a very basic example. We'll use the computer object as a base to make it easier to test.
+
+A note about file location. This can be placed anywhere you'd put other DM code. The frontend must be placed in a specific place - more on that below.
 
 ```dm
-/obj/machinery/my_machine/ui_interact(mob/user, datum/tgui/ui)
+/obj/machinery/computer/TGUI_Test_Computer
```
Comment (Fikou) [inline tgui/docs/tutorial-and-examples.md:44]: whys this capitalized

**Comment by stylemistake** [comment]
> Uses codeblocks

This will fix issues with mixed tab/spaces across the project: https://github.com/editorconfig/editorconfig-codeblocks

**Comment by remove32** [comment]
> 
> 
> > Uses codeblocks
> 
> This will fix issues with mixed tab/spaces across the project: https://github.com/editorconfig/editorconfig-codeblocks. I recommend using VSCode though.

oh god no i dont use code::blocks I just mean the ``` sections

File: tgui/docs/tutorial-and-examples.md (hunk 1 above)
Comment (remove32) [reply tgui/docs/tutorial-and-examples.md:44]: because i'm a menace fikou
(i'll fix it next pass)

**Pull request closed without merging**